pkg/values/big_int.go

```diff
@@ -45,6 +46,10 @@ func (b *BigInt) UnwrapTo(to any) error {
 		}
 		*tb = b.Underlying
 	default:
+		rto := reflect.ValueOf(to)
+		if rto.CanConvert(reflect.TypeOf(new(big.Int))) {
```

Comment:
what is special about big.Int in the existing system? why aren't there other branches in this `if`?

Reply:
This is in the class for *big.Int, if you can cast to a big int then you're a big int. If you can't then it's a problem.

Reply:
ah, thank, i missed that